moped-editor/src/theme/index.js

```diff
@@ -27,7 +27,7 @@ const theme = createMuiTheme({
     },
   },
   shadows,
-  typography,
+  // typography,
```

Comment:
If topography does not need to be there, then best to remove the comments too.

Reply:
agreed